traveler-ios-ui/TravelerKitUI/UIUtilities/HeartButton.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  HeartButton.swift
+//  TravelerKitUI
+//
+//  Created by Ben Ruan on 2019-09-16.
+//  Copyright © 2019 GuestLogix. All rights reserved.
+//
+
+import Foundation
+
+@IBDesignable class HeartButton: UIButton {
+    @IBInspectable var strokeWidth: CGFloat = 1.5
+
```

Comment:
no extra line....group variables together.

code has to look good and consistent